pkgs/servers/caddy/default.nix

```diff
@@ -16,9 +16,13 @@ buildGoModule rec {
   };
   modSha256 = "02cb3swc180kh5vc2s5w8a6vidvw768l9bv5zg8zya183wzvfczs";
 
-  buildFlagsArray = ''
-    -ldflags=
-      -X github.com/mholt/caddy/caddy/caddymain.gitTag=v${version}
+  buildFlagsArray = [
+    "-ldflags=" "-w -s -X github.com/mholt/caddy/caddy/caddymain.gitTag=v${version}"
+  ];
+
+  preBuild = ''
+    substituteInPlace caddy/caddymain/run.go \
+      --replace "var EnableTelemetry = true" "var EnableTelemetry = false"
```

Comment:
Can you make this a patch instead?

Reply:
also nitpick: `var EnableTelemetry = false` is the same as `var EnableTelemetry bool`.